app/views/layouts/mailer.text.erb

```diff
@@ -0,0 +1 @@
+<%= yield %>
```

Comment:
...or this

Reply:
Do we need to remove from gov template as well as from register?